graphql/schema/types/version.graphql

```diff
@@ -49,6 +49,7 @@ type Version {
   errors: [String!]!
   externalLinksForMetadata: [ExternalLinkForMetadata!]!
   finishTime: Time
+  generatedTaskCounts: Map!
```

Comment:
I slightly wonder if we should do something similar to the `StringMap` scalar and create another scalar so that the resolver can return `map[string]int`. It's not a big deal overall though